sky/engine/core/script/dart_init.cc

```diff
@@ -341,6 +328,74 @@ void* _DartSymbolLookup(const char* symbol_name) {
   return DartLookupSymbolInLibrary(symbol_name, nullptr);
 }
 
+#elif OS(ANDROID)
+
+// Describes an asset file that holds a part of the precompiled snapshot.
+struct SymbolAsset {
+  SymbolAsset(const char* symbol, const char* file, bool exec)
+      : symbol_name(symbol), file_name(file), is_executable(exec),
+        mapping(nullptr) {};
```

Comment:
I probably would have skipped this constructor and just used initializer syntax, but I think the compiler is smart enough to figure that out.